prboom2/src/MUSIC/dbopl.c

```diff
@@ -1249,53 +1249,19 @@ void Chip__WriteReg(Chip *self, Bit32u reg, Bit8u val ) {
   }
 }
 
-Bit32u Chip__WriteAddr(Chip *self, Bit32u port, Bit8u val ) {
```

Comment:
Might want to compare against upstream Dosbox OPL code and see if these are also unused there or there's missing emulation features that should be merged here.

Reply:
I'd rather preserve this file as much as possible for now.